manifests/y/yourmoln/moligeek/0.6.4/yourmoln.moligeek.installer.yaml

```diff
@@ -3,12 +3,17 @@
 
 PackageIdentifier: yourmoln.moligeek
 PackageVersion: 0.6.4
-Installers:
-- Architecture: x64
-  InstallerType: portable
-  InstallerUrl: https://github.com/yourmoln/moligeek/releases/download/0.6.4/moligeek.exe
-  InstallerSha256: 91FF053982CEE4E9C7883EFDCCCC9BB7AC7B8757D48759391599322C08B39644
-  Commands:
+Commands:
   - moligeek
+Installers:
+  - Architecture: x64
+    InstallerType: portable
+    InstallerUrl: https://github.com/yourmoln/moligeek/releases/download/0.6.4/moligeek.exe
+    InstallerSha256: 91FF053982CEE4E9C7883EFDCCCC9BB7AC7B8757D48759391599322C08B39644
+  - Architecture: x64
+    InstallerType: portable
+    InstallerLocale: zh-CN
+    InstallerUrl: https://gitee.com/yourmoln/moligeek/releases/download/0.6.4/moligeek.exe

```

Comment:
Reviewing both the GitHub and Gitee page shows they have the same content - both indicate installing from GitHub without referencing Gitee at all. Were it the other way around, and GitHub referenced installing from Gitee, it would be approved.

Reply:
CC @yourmoln